dnit.ts

```diff
@@ -261,9 +261,9 @@ export class Task {
     if (actualUpToDate) {
       ctx.taskLogger.info(`--- ${this.name}`);
     } else {
-      ctx.taskLogger.info(`... ${this.name}`);
+      ctx.taskLogger.info(`{ ${this.name}`);
```

Comment:
Completely minor thing, but perhaps 3 character wide strings   `{{{` and `}}}` or `{--` and `--}'  etc so that text lines up with up to date tasks?

Reply:
Ha ha actually was the point of the review - I was sure someone would have an opinion on one vs 3 brackets.